contracts/utils/Filters.sol

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.9;
+
+import { IndexMap, LibIndexMap } from "../libraries/LibIndexMap.sol";
+
+/**
+ * @dev Checks if the given `index` is not present in the `nonRotatingMap`.
+ * @param index The index to check.
+ * @param nonRotatingMap The non-rotating map to search in.
+ * @return A boolean value indicating whether the `index` is not present in the `nonRotatingMap`.
+ */
+function nonStandardValidatorFilter(uint256 index, uint256[] memory nonRotatingMap) pure returns (bool) {
```

Comment:
Please rename it to "what this method actually does", not "what this method is for".
E.g. `notInRotatingMapFilter`

Reply:
Same for the others. And please, revise all the docstring.

Reply:
renamed.